reactor-core/src/test/java/reactor/core/publisher/FluxRetryWhenTest.java

```diff
@@ -69,7 +79,7 @@ public void cancelsOther() {
 		Flux<Integer> when = Flux.range(1, 10)
 		                         .doOnCancel(() -> cancelled.set(true));
 
-		StepVerifier.create(justError.retryWhen(other -> when))
+		StepVerifier.create(justError.retryWhen(() -> other -> when))
```

Comment:
since we're changing the uses, we should ensure that the old APIs are still working. Half way thought the review, ignore if we have tests for that :)

Reply:
yes there's a smoke test of the APIs to show that old use works